include/config/tablet.h

```diff
@@ -0,0 +1,12 @@
+/* SPDX-License-Identifier: GPL-2.0-only */
+#ifndef LABWC_TABLET_H
+#define LABWC_TABLET_H
+
+#include <stdint.h>
+
+uint32_t tablet_button_from_str(const char *button);
+uint32_t mouse_button_from_str(const char *button);
+void add_tablet_button_mapping(uint32_t from, uint32_t to);
```

Comment:
We try to start extern function names with the filename of the translation unit, so `tablet_` in this case.

Suggest:
- `tablet_button_mapping_add()`
- `tablet_load_default_button_mappings()` (seems consistent with mouse/keyboard binds in `post_processing()` in `rcxml.c`)

`mouse_button_from_str()` is probably not easy to 'tabletize', so happy to leave it.

Reply:
Thanks, I took over your suggestions.